src/datatypes/character/CharacterDescription.hpp

```diff
@@ -19,30 +20,41 @@ namespace spy::character {
         DIVERSE
     };
 
+    NLOHMANN_JSON_SERIALIZE_ENUM(GenderEnum , {
+        { GenderEnum::MALE, "MALE" },
+        { GenderEnum::FEMALE, "FEMALE" },
+        { GenderEnum::DIVERSE, "DIVERSE" },
+    })
+
```

Comment:
dieses makro bewirkt, dass bei einem fehler beim parsen das erste element genommen wird. Ist das hier problematisch? Ich habe an anderer Stelle schon mal so ein "INVALID" eintrag ins enum genommen, ist aber halt nicht besonders schön... Wegen mir kann man das gerne hier so lassen und dann mal ein issue aufmachen dass man das vereinheitlicht.

https://github.com/nlohmann/json#specializing-enum-conversion

Reply:
Aktuell ist auch noch nicht berücksichtigt, dass z.B. keine Koordinate in der json sein könnte.

Reply:
Habe hier DIVERSE als default gemacht. Gender ist ja so oder so nur für GUI wichtig